packages/react/src/components/ResourceList/__snapshots__/ResourceList.story.storyshot

```diff
@@ -305,7 +313,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/R
             aria-describedby={null}
             aria-pressed={null}
             className="iot--btn bx--btn bx--btn--ghost"
-            data-testid="Button"
```

Comment:
These button changes look like breaking changes, but were only added in an earlier testId PR. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/206bcede76ef83c512b11b9aff3e9548b079b0f5/packages/react/src/components/ResourceList/__snapshots__/ResourceList.story.storyshot